README.md

```diff
@@ -5,94 +5,140 @@
 
-### Requirements
-- Java 8 or 11
-- maven 3.3.9 or later
-- as of version 2.6.1 Spring Boot >= 2.2 is required. Consequently, it also requires Spring Security version >= 5.2
+In the table below you'll find links to detailed information.
 
-### Sample
-- See [spring-security-hybrid-usage](samples/spring-security-hybrid-usage) for an example.
-- See [spring-security-basic-auth](/samples/spring-security-basic-auth) for an example demonstrating how a user can access Rest API via basic authentication (user/password) using [spring-xsuaa](./spring-xsuaa).
+| Library                                   | Usage Examples                                                                                                                                                                                                                                         | 
+|-------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [token-client](/token-client)             | [java-tokenclient-usage](samples/java-tokenclient-usage) demonstrates usage of token client library in Java EE application<br/>   [spring-security-xsuaa-usage](samples/spring-security-xsuaa-usage) demonstrates usage in Spring Boot application     |              
 
-### Additional (test) utilities
-- [java-security-test](./java-security-test) offers test utilities to generate custom JWT tokens for the purpose of tests. It pre-configures a [WireMock](http://wiremock.org/docs/getting-started/) web server to stub outgoing calls to the identity service (OAuth resource-server), e.g. to provide token keys for offline token validation. Its use is only intended for JUnit tests.
 
+## Installation
+The SAP Cloud Security Services Integration is published to maven central: https://search.maven.org/search?q=com.sap.cloud.security and is available as a Maven dependency. Add the following BOM to your dependency management in your `pom.xml`:
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.sap.cloud.security</groupId>
+            <artifactId>java-bom</artifactId>
+            <version>3.0.0</version>
+            <scope>import</scope>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+```
+along with libraries that you intend to use e.g. `java-security`
+```xml
+<dependencies>
+    <dependency>
+        <groupId>com.sap.cloud.security</groupId>
+        <artifactId>java-security</artifactId>
+    </dependency>
+</dependencies>
+```
+:bulb: Please refer to each library's that you intend to use `README.md` Maven Dependencies section for detailed information on which dependencies need to be added to the `pom.xml`.
```

Comment:
Hard to read this sentence on first attempt.
Suggestion:
Please refer to the Maven Dependencies section in the `README.md` of the library you intend to use for detailed [...]